extensions/shared/components/block-styles-selector/index.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * External Dependencies
+ */
+import classnames from 'classnames';
+import { memo } from 'React';
```

Comment:
This is breaking things:

```
Module not found: Error: Can't resolve 'React' in '/home/travis/build/Automattic/jetpack/extensions/shared/components/block-styles-selector'
```

Can we do without all of React here? Can we get it from `@wordpress/element` instead?

Reply:
There is a `useMemo` HOC in `@wordpress/element`, but when I use this I get an error from React. Looking into it some more...

Reply:
Ok I added a commit to take care of this too.

Reply:
We might be able to change this to `useMemo` but we need `useSelect` which (as you've seen) also isn't available. It's a shame, but treating it like we have for the changes to `BlockPreview` makes sense to me.